### DISCUSSION TITLE
Happy Family Day!!

### DISCUSSION THREAD

**Discussion by SudharakaP** [opencollective/opencollective#3979] [General]
Today is [Family Day](https://en.wikipedia.org/wiki/Family_Day) in Canada! Hope ya'll have a wonderful time with your family and friends! 😄 

![image](https://user-images.githubusercontent.com/12435965/107977254-a843ca00-6f6f-11eb-9c70-9b4845ebcc06.png)

**Comment by Betree** [comment]
I didn't know about Family day (it doesn't exist in France). Happy Familly day to you too!
